tests/Doctrine/Tests/ORM/UnitOfWorkTest.php

```diff
@@ -150,7 +150,29 @@ public function testChangeTrackingNotify()
         $this->_unitOfWork->persist($entity);
 
         $this->_unitOfWork->commit();
-        $this->assertEquals(1, count($persister->getInserts()));
+        $this->assertCount(1, $persister->getInserts());
+
+        // Create and Set first entity
+        $entity1 = new NotifyChangedEntity;
```

Comment:
These assertions should be moved to their own test case, with an `@group 5579` annotation to identify the issue it's related to